src/Admin.php

```diff
@@ -25,4 +25,89 @@ public static function admin_notices() {
     <?php }
   }
 
+  /**
+   * Returns products backlinking to a given accessory product.
+   */
+  public static function wp_ajax_related_accessories_backlinks() {
+    global $wpdb;
+    $postId = intval($_POST['post']);
+    $html = '<ol>';
+    if($postId)
+    {
+      $params = [
+        'post_type' => ['product', 'product_variant'],
+        'post_status' => 'any',
+        'meta_query' => [
+          ['key' => 'field_group_related_accessories_care_products',
+            'value' => '"' . $postId . '"',
+            'compare' => 'LIKE',
+          ],
+        ],
+        'posts_per_page' => -1,
+      ];
+      $wc_query = new \WP_Query($params);
+      global $post, $product;
+      if ($wc_query->have_posts()) {
+        while ($wc_query->have_posts()) {
+          $wc_query->the_post();
+          $html .= '<li><a href="'.get_edit_post_link($wc_query->post).'">';
+          $html .= esc_html($wc_query->post->post_title);
+          $html .= '</a></li>';
+        }
+        $html .= '</ol>';
+      }
+      else $html = 'No results';
```

Comment:
Coding standards. And borderline PHP parse error - I didn't even know that this syntax is allowed.
```suggestion
      else {
        $html = 'No results';
      }
```

Reply:
And missing translation.
```suggestion
      else {
        $html = __('No results', Plugin::L10N);
      }
```